src/overlays/actors/ovl_En_Encount3/z_en_encount3.h

```diff
@@ -10,7 +10,28 @@ typedef void (*EnEncount3ActionFunc)(struct EnEncount3*, PlayState*);
 typedef struct EnEncount3 {
     /* 0x000 */ Actor actor;
     /* 0x144 */ EnEncount3ActionFunc actionFunc;
-    /* 0x148 */ char unk_148[0x84];
+    /* 0x148 */ u8 unk148;
+    /* 0x149 */ char pad149[1];
```

Comment:
This looks like padding due to alignment of the types, you should be able to remove it